website/source/docs/commands/index.html.md

```diff
@@ -37,6 +37,7 @@ Available commands are:
     keyring        Manages gossip layer encryption keys
     kv             Interact with the key-value store
     leave          Gracefully leaves the Consul cluster and shuts down
+    license        Get/Put the Consul Enterprise license (Enterprise-only)
```

Comment:
I don't see this "(Enterprise-only)" marking elsewhere (such as `consul --help`) and it doesn't show up in OSS (as you would expect!). Should we drop it to keep this matching the actual output?

Reply:
Ah I see. I didn't see it on `snapshot --help` for the agent either (I believe that is ent only) so I guess we're just not consistent. Definitely no big deal here either way, I'm cool with leaving it for clarity.